README.md

```diff
@@ -110,74 +108,55 @@ Note on AIX Support:
 
 ### Configure node
 
-Once the cookbook is available in Chef Server, you need to add the `audit::default` recipe to the run-list of each node. The profiles are selected via the `node['audit']['profiles']` attribute. A complete list of the possible configuration are documented in [Supported Configurations](docs/supported_configuration.md). For example you can define the attributes in a role or environment file like this:
+Once the cookbook is available in Chef Server, you need to add the `audit::default` recipe to the run-list of each node. The profiles are selected via the `node['audit']['profiles']` attribute. A list of example configurations are documented in [Supported Configurations](docs/supported_configuration.md). Below are some other examples:
 
+```ruby
+default['audit']['reporter'] = 'chef-server-compliance'
 
-node.default['audit']['profiles'].push("path": "#{PROFILES_PATH}/mylinux-failure-success")
```

Comment:
Maybe multiple `.push({profile_hash})` would better than `= [array, of, profile, hashes]` (below) so it's additive instead of clobbering?

Reply:
Will add it to the other examples though!